oci/private/tarball.sh.tpl

```diff
@@ -7,11 +7,6 @@ readonly BLOBS_DIR="{{blobs_dir}}"
 readonly TAGS_FILE="{{tags}}"
 readonly TARBALL_MANIFEST_PATH="{{manifest_path}}"
 
-REPOTAGS=()
-# read repotags file as array and prepend it to REPOTAGS array.
-IFS=$'\n' REPOTAGSFILE=($(cat "$TAGS_FILE"))
-REPOTAGS=(${REPOTAGSFILE[@]+"${REPOTAGSFILE[@]}"} ${REPOTAGS[@]+"${REPOTAGS[@]}"})
```

Comment:
@alexeagle @aryeh-looker While testing I also figured out that this logic did not work with multiple repotags -- it would only store the first one in the REPOTAGS variable.

Given that the tags file was already space separated, all I had to do was read it into the REPOTAGS variable directly because the IFS is space by default:
```
REPOTAGS=$(cat $TAGS_FILE)
```

This in combination with my fix below allowed me to create an image from a tarball with multiple repotags:
```
[ 
    { 
         "Config": "blobs/sha256/300fcd932ebbfab3377f4d1450e2e610ecabbbebda1dc6166ab3bc1b18414985",
         "RepoTags": [ "phono:latest", "test:latest" ], 
         "Layers": [
            "blobs/sha256/6b0b1527d75b1e49c2cb08f4eb8616c64c9a20a8b9d8c1179146b2675586536b.tar.gz",   
            "blobs/sha256/26e3e4b0848c0e87391a7a844b11fcd2e6d223bd516be6ac8b17bac4e6808609.tar.gz", 
            "blobs/sha256/336c487350cddcf412379144afef9375e4e4611df01abc41ab558e84bbcb80b8.tar.gz" 
         ] 
    } 
]
```

Reply:
> Given that the tags file was already space separated

they are supposed to be `newline` separated

Reply:
Well something else is likely broken then, because when I logged the file contents they were space separated 😕

Reply:
depends how you produced the file, but if you use the `oci_tarball` macro it puts one-per-line here https://github.com/bazel-contrib/rules_oci/blob/main/oci/defs.bzl#L102

Reply:
@alexeagle I verified that it is newline separated, but the bash script is not correctly handling the newlines.
I tried this locally:
```
echo "$(cat $TAGS_FILE)"

REPOTAGS=()
# read repotags file as array and prepend it to REPOTAGS array.
IFS=$'\n' REPOTAGSFILE=($(cat "$TAGS_FILE"))
for repotag in $REPOTAGSFILE
do
  echo "$repotag"
done
```

And this was the output:
```
phono:latest
test:latest
phono:latest
```

I.e. the file does have the newlines but the REPOTAGSFILE is losing all of the lines after the first.